dom-event-delegation/main.js

```diff
@@ -0,0 +1,13 @@
+const $taskList = document.querySelector('.task-list');
+$taskList.addEventListener('click', taskListClick);
+
+function taskListClick(event) {
+  console.log('event target:', event.target);
+  console.log('target tag name: ', event.target.tagName);
+  const $button = document.querySelector('button');
+  if (event.target === $button) {
```

Comment:
You shouldn't be querying for the button here, you should be checking if the `tagName` is a button